addons/base_import/models/base_import.py

```diff
@@ -542,35 +556,37 @@ def _remove_currency_symbol(self, cr, uid, value):
             return False
         if len(split_value) == 1:
             if float_regex.search(split_value[0]) is not None:
-                return split_value[0] if not negative else '-'+split_value[0]
-            return False 
+                return split_value[0] if not negative else '-' + split_value[0]
+            return False
         else:
             # String has been split in 2, locate which index contains the float and which does not
             currency_index = 0
             if float_regex.search(split_value[0]) is not None:
                 currency_index = 1
             # Check that currency exists
-            currency = self.pool.get('res.currency').search(cr, uid, [('symbol', '=', split_value[currency_index].strip())])
+            currency = self.env['res.currency'].search([('symbol', '=', split_value[currency_index].strip())])
             if len(currency):
```

Comment:
Might as well `limit=1` on the search while at it, and check `if currency`, no need for the `len`.